src/App.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
+import { Key } from './component/Component';
 
 export const App: React.FC = () => (
   <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
+    <Key key={0} />
```

Comment:
The 'key' prop should not be used here because it's a reserved prop in React and its purpose is to help with list rendering. Use a different prop name to pass the last pressed key value.